### DISCUSSION TITLE
Getting npm ERR! code TAR_BAD_ARCHIVE and npm ERR! TAR_BAD_ARCHIVE: Unrecognized archive format

### DISCUSSION THREAD

**Discussion by Dilipkumarkanta** [community/community#147302] [npm]
I am getting the below error when i am doing npm install
npm ERR! code TAR_BAD_ARCHIVE
npm ERR! TAR_BAD_ARCHIVE: Unrecognized archive format

Here is the docker file for your referance.
FROM node:18.19.1
#Working directory
WORKDIR /app

# Install the timezone package and copy the specific Time Zone file to etc.
#RUN apk add --no-cache tzdata && cp /usr/share/zoneinfo/Asia/Kolkata /etc/localtime

# Copy the cloned repo to the working directory.
COPY ./ .

#Install Node Modules
RUN npm cache clean --force
RUN npm install --force

#Createing the build folder
RUN npm run build

#Application port
EXPOSE 3000

# Set BROWSER environment variable to none
#ENV BROWSER=none

CMD ["npm", "run", "start"]

**Comment by xtroncode** [comment]
https://github.com/npm/cli/issues/7986

**Reply by Dilipkumarkanta** [reply]
Thanks you very much

**Reply by Dilipkumarkanta** [reply]
We have one more issue can their any discussions going on the issue is below.

#16 [frontend_dependencies  9/11] RUN npm install -g npm@latest
#16 2.272 npm ERR! code EBADENGINE
#16 2.273 npm ERR! engine Unsupported engine
#16 2.273 npm ERR! engine Not compatible with your version of node/npm: npm@11.0.0
#16 2.274 npm ERR! notsup Not compatible with your version of node/npm: npm@11.0.0
#16 2.274 npm ERR! notsup Required: {"node":"^20.17.0 || >=22.9.0"}
#16 2.274 npm ERR! notsup Actual:   {"npm":"10.2.4","node":"v20.11.1"}
